src/_mixins.scss

```
@@ -142,6 +142,14 @@ $toplevel-border-color: rgba(black, 0.2);
     }
 }
 
+@function etched-shadow() {
+    @if $color-scheme == "dark" {
+        @return 0 -1px 1px rgba(black, 0.67);
```

This probably needs to be a shade of the primary color so that you don't get muddy colors with greens and yellows. Something like
```suggestion
        @return 0 -1px 1px #{'shade (@color_primary, 0.75)'};
```

Which also really only makes sense in titlebars